src/Data/Profunctor/Types.hs

```diff
@@ -136,11 +171,26 @@ instance Monad (Costar f a) where
   return = pure
   Costar m >>= f = Costar $ \ x -> runCostar (f (m x)) x
 
+instance Comonad f => Category (Costar f) where
+  id = Costar extract
+  {-# INLINE id #-}
+  Costar f . Costar g = Costar (f =<= g)
+  {-# INLINE (.) #-}
+
 ------------------------------------------------------------------------------
 -- Wrapped Profunctors
 ------------------------------------------------------------------------------
 
--- | Wrap an arrow for use as a 'Profunctor'.
+-- |
+-- - 'Arrow' is equivalent to 'Category'
+--   && t'Data.Profunctor.Strong.Strong'.
+-- - 'ArrowChoice' is equivalent to 'Category'
+--   && t'Data.Profunctor.Strong.Strong' && t'Data.Profunctor.Choice.Choice'.
+-- - 'ArrowLoop' is equivalent to 'Category'
+--   && t'Data.Profunctor.Strong.Strong' && t'Data.Profunctor.Strong.Costrong'.
+--
+-- This newtype allows 'Profunctor' classes to be used with types that only
+-- implement @base@'s arrow classes.
```

Comment:
These Haddocks are now somewhat indirect, as they begin with a list of relationship between `base` classes and `profunctor` classes, and then they are followed with an explanation of what the `WrappedArrow` type actually is. I think it should be the other way around: first say what `WrappedArrow` is, and then use that to motivate the relationships between the classes.